### DISCUSSION TITLE
New install getting Error Code 500

### DISCUSSION THREAD

**Discussion by stevedngo** [immich-app/immich#953] [Q&A] [answered]
I just installed Immich in Portainer and when I tried accessing the web server I'm getting the following message
Error code 500
Internal Error
Verbose
{"message":"Internal Error"}

![Screenshot 2022-11-09 at 11 40 06 PM](https://user-images.githubusercontent.com/17154898/201029143-d9f93055-f310-475f-a0c0-e984f96e8a79.png)

**Comment by bo0tzz** [comment]
Can you check the logs of the immich-web and immich-server containers to see if they have any errors?

**Comment by stevedngo** [comment, marked as answer]
I got it working after I checked the server log and it showed password authentication failed for user. I edited my .env to use the default one. Not sure why I couldn't use my own info.
